external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -181,40 +203,12 @@ export type TraceEvent =
         name: string,
         modInfo: ModuleInfo,
         localsTypes: string[],
-        paramValues: TraceValue[]
+        paramValues: RuntimeValueType[]
     }
     | { type: TraceEventKind.CloseFrame, id: number }
     | { type: TraceEventKind.Instruction, pc: number, kind: TraceInstructionKind }
     | { type: TraceEventKind.Effect, effect: EventEffect };
 
-/**
```

Comment:
Replaced trace-level location with runtime-level location as we need it now in the runtime as well and there is no good reason for duplication here